src/main/java/com/snowplowanalytics/snowplow/tracker/payload/TrackerPayload.java

```diff
@@ -40,11 +40,11 @@
     @Override
     public void add(final String key, final String value) {
         if (key == null || key.isEmpty()) {
-            LOGGER.error("Invalid key detected: {}", key);
+            LOGGER.error("Invalid key detected");
```

Comment:
Is this more useful?

```suggestion
            LOGGER.error("Invalid key detected. Key is null or empty");
```